source/Calamari.Shared/Integration/Packages/Download/GitHubPackageDownloader.cs

```diff
@@ -238,6 +238,14 @@ JToken PerformRequest(ICredentials feedCredentials, string uri)
             }
         }
 
+        void AddAuthorization(WebClient client, string? authorizationToken)
+        {
+            if (!String.IsNullOrWhiteSpace(authorizationToken))
+            {
+                client.Headers.Set(HttpRequestHeader.Authorization, String.Concat("token ", authorizationToken));
```

Comment:
This is the actual fix - explicitly setting the `Authorization` header to use the OAuth/Personal Access token